x/docs/md/cli/mac.md

```diff
@@ -1,122 +1,80 @@
-didn't.
+1. <a href="#troubleshooting">troubleshooting for instructions on installing without Homebrew</a>
+2. [exercism.io general help](http://exercism.io/help)
+3. [join the exercism.io chat on gitter](https://gitter.im/exercism/support): [![Join the chat at https://gitter.im/exercism/support](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/exercism/support)
 
-Find out if you have homebrew installed via the terminal.
+### Installing With Homebrew   
 
-You can open a terminal using Spotlight by pressing <command><spacebar> and
-then type 'terminal' in the space provided.
+#### 1: Check For Homebrew
 
+Homebrew is a package manager for OS X which installs the stuff that Apple didn't.
+Find out if you have homebrew installed via the terminal.
+You can open a terminal using Spotlight by pressing <spacebar> and then type 'terminal' in the space provided.
 On the command prompt, type in the command:
 
 `\brew --version`
 
 If homebrew is installed you may see output like the following (version numbers may vary):
-
 ```
 Homebrew 0.9.9 (git revision a5586; last commit 2016-05-09)
 Homebrew/homebrew-core (git revision 3b4c; last commit 2016-05-09)
 ```
+If homebrew isn't installed, you can: 
+1. install via [homebrew's brew.sh site](http://brew.sh/)
+2. see [troubleshooting for instructions on installing without homebrew](#troubleshooting)
 
-Install the CLI via homebrew in the following section.  Otherwise see
-"[Installing Without Homebrew](#installing-without-homebrew)".
-
-### Installing With Homebrew <a name="installing-with-homebrew"></a>
-
-Install the CLI via [homebrew](http://brew.sh/) with the following command:
+#### 2: Homebrew Install 
+Install the CLI via homebrew with the following command:
 
 ```
-brew update && brew install exercism
+brew update && brew install exercism 
 ```
 
 Verify that it was installed properly by running:
 
-```bash
-exercism --version
 ```
-
-If there was a problem you will get an error message saying _command not found_.
-
-If everything is fine, you're done installing the CLI, and you can move on to the next step,
-which is configuring the CLI.
-
-### Installing Without Homebrew <a name="installing-without-homebrew"></a>
-
-This is described in detail in [this video tutorial](https://www.youtube.com/watch?v=TCT4eHGwfaE).
-
-First you need to know which processor architecture your computer has. If
-you're not sure, you can use Terminal.app to find out:
-
-Open Terminal.app and type in the following command:
-
-```bash
-uname -m
+exercism --version 
 ```
 
-Common values are `i386` (32-bit) and `x86_64` (64-bit). If you have something
-different, you'll need to ask Google about the details.
+If there was a problem you will get an error message saying command not found.
 
-If you don't have a directory in your home directory called `bin`, make one now:
+#### 3: Verifying Your Installation 
+Verify that the binary was installed properly by running:
 
-```bash
-mkdir ~/bin
 ```
```

Comment:
This line break doesn't belong here. `bash` should be right after the three ticks. This is wrong in various places.